### ISSUE TITLE
yle fem/areena

### ISSUE THREAD

**Issue by jussijjj** [hirsivaja/plugin.video.areena#23]
Need a plugin update because the channel merge

**Comment by Huxely** [comment]
My live streams stopprd working altogether. This happened around same time I updated my libreelec, so kodi version went from 17.0 to 17.3, but I downgraded and still they don't work. Is this related to the channel rearrangement?

**Comment by raspikodi4ever** [comment]
True! Also strange behavior when launching available areena videostreams in Kodi Krypton 17.3, LibreElec 8.0.2, when user clicks a program, seems that nothing happens, user has to manually go to full screen mode, while screen remains black, must browse back and forth in the open stream to get the playback start. Please fix these issues, would like to keep watching Areena in Rpi Kodi.

Also download of programs would be desirable option for Rpi users, to whom Flick Fetch isn't an option.

**Comment by hirsivaja** [comment]
I could not find a direct url for the new TeemaFem channel so I just removed the non-working channels from the list. If somebody knows the url please comment here.

Instead I added the channels using the Yle API. But unfortunately it has the same limitations as before and not all live content is available.

**Comment by T3m3z** [comment]
@hirsivaja This might be the link for Yle Teema/Fem: http://yletv-lh.akamaihd.net/i/yleteemafemfi_1@490775/master.m3u8

**Comment by hirsivaja** [comment]
Thanks @T3m3z. I added the channel to the list in 3cda7a0ae887281ae5274a6e5ba9afb22e3537a5
